roles/openshift_health_checker/test/openshift_check_test.py

```diff
@@ -81,6 +81,7 @@ class TestCheck(OpenShiftCheck):
 @pytest.mark.parametrize("keys,expected", [
     (("foo",), 42),
     (("bar", "baz"), "openshift"),
+    (("bar.baz",), "openshift"),
```

Comment:
Why not? It's two extra lines and it matches what we do elsewhere in ansible code. Personally I would like the dotted version better but I don't want to go through the code and change them all.

I'm not as set on the "single obvious way to do it" ethos of python...